source/dea_seq_reset.rst

```diff
@@ -29,12 +29,35 @@ How is this Anomaly Diagnosed?
 * The science run will be terminated, sending a ``scienceReport`` packet that will contain
   non-zero ``fepErrorCodes`` and a non-zero ``terminationCode``.
 * Within a major frame (32.2 seconds), there will be a slight drop in DPA Input Current A and/or B,
-  (1DPICACU and/or 1DPICBCU) ~0.2-0.3 A. The drop in two input currents depends on which side is
-  running which FEPs.
+  (1DPICACU and/or 1DPICBCU) ~0.2-0.3 A. The drop in two input currents depends on which FEPs
+  were active (DPA Side A runs FEPs 0, 1, and 2; DPA side B runs FEPs 3, 4, and 5.)
 * Within a major frame, the 1STAT1ST bilevel will toggle off (science idle).
 * DEA Housekeeping will show sharp drops in the temperatures of FEP 0 and/or FEP 1, if they are
   running.
 
+If only some of the FEPs were halted and the science run continues, this might be a
+case of :ref:`fep_reset`.
```

Comment:
sorry, yes, backticks around the ../fep_reset part. unfortunately it doesn't render correctly in here.